db/migrations/versions/a0b9e8f859a9_.py

```diff
@@ -0,0 +1,32 @@
+"""empty message
+
+Revision ID: a0b9e8f859a9
+Revises: 5bf853808dfb
+Create Date: 2022-11-01 16:53:20.200347
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "a0b9e8f859a9"
+down_revision = "5bf853808dfb"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table("applications", schema=None) as batch_op:
+        batch_op.add_column(sa.Column("language", sa.String(), nullable=False))
```

Comment:
This isn't going to work when deployed, as existing columns will be `null` i.e. they won't be `en` (as we've set currently in the test data.)

Instead, I think this needs to be `nullable=True`, and in the application code, assume `null` to be `en`